contrib.md

```diff
@@ -0,0 +1,46 @@
+# Contributing
+
+When contributing to this repository, please create a pull request.  If you would like to have interest
+of an idea guaged, please create an issue or request feedback in the flow projects slack workspace.
+
+## Development
+
+For making it easier to create pull requests, you can use the below commands to install the components in scratch orgs for development purposes.
+
+You must install [cumulus ci](https://cumulusci.readthedocs.io/en/stable/intro.html) to run the below commands. You can set up a scratch org with the components in this repository by running the below command:
```

Comment:
Thanks @jmdohn. I fear I may not have understood your thought process. Here's the issue: this text can be thought of as the path that a new prospective contributor might follow. It's a steep path and I want to encourage new contributors. The problem with inserting cumulus ci stuff here is that most new contributors won't have experience with ci/cd or cumulus, so they'll have to puzzle through this to figure out that it's optional.  So what I'd like to do is decouple the cumulus stuff into a separate document so that the path of new contributor just sees something like "if you are using cumulus ci, click here for some useful utilities". Thoughts?

Reply:
I'm hoping to automate many of the PR requested steps with github actions so they don't need to remember / do the steps below for adding a "net new or archiving an existing" component and auto adding it to the appropriate action group based on directory for cci. Although, this will take at least a few weeks to complete depending on my bandwidth. I definitely appreciate the need to keep it low level of entry to folks who may not be familiar with cci and am totally okay with breaking it out into a separate document and referencing that in the readme.

Happy to also add details in that same document for those using just sfdx for their work.

Once I get some of the more fancy CCI automation done, I'm happy to take some time to run you through it before creating that PR because it's some pretty geeky stuff that most contributors won't need to interact with in nearly every circumstance (unless they really really want to).